### ISSUE TITLE
[Bug] error handling in stocks/fa/dupont

### ISSUE THREAD

**Issue by jmaslek** [OpenBB-finance/OpenBBTerminal#5087]
Found from logs:

```
2023 May 31, 16:36 (🐛) /stocks/fa/ $ dupont -t DRS

Error: Length of values (0) does not match length of index (6)

Error: 'list' object has no attribute 'empty'
```

**Comment by deeleeramone** [comment]
It's also unnecessary for this function to be "exclusive" to the AlphaVantage data. It is only taking standard line items from 10K/Q forms and doing basic math.  There is nothing unique to the source here.

**Comment by colin99d** [comment]
Fixed by #5147
